server/controllers/userController.js

```diff
@@ -32,29 +34,41 @@ userController.createUser = async (req, res, next) => {
     const userValues = [
       first_name,
       last_name,
-      first_name + ' ' + last_name,
+      first_name + " " + last_name,
       username,
-      email, 
+      email,
+      company,
       hashedPassword,
-      is_landlord
+      is_landlord,
     ];
     const userResult = await db.query(queries.createUser, userValues);
-    delete userResult.rows[0].password;
-    res.locals.user = userResult.rows[0];
+
+    //handle landlord creation
```

Comment:
Should be handled in landlord controller to maintain independence between landlord and user creation

Reply:
fixed!